manifests/python.yml

```diff
@@ -1102,7 +1102,7 @@ tests/:
       Test_TracerSCITagging: v1.12.0
       Test_TracerUniversalServiceTagging: v0.36.0
     test_tracer_flare.py:
-      TestTracerFlareV1: missing_feature (failure 2.8.0)
+      TestTracerFlareV1: v3.12.0.dev
```

Comment:
Can we find the minimum tracer version that supports tracer flares. This will improve the reporting in the FPD.